examples/virtio/build.zig.zon

```diff
@@ -0,0 +1,17 @@
+.{
+    .name = "virtio",
+    .version = "1.0.0",
+
+    .dependencies = .{
+        .libvmm = .{
+            .path = "../../"
+        },
+        .sddf = .{
+            .path = "../../dep/sddf"
+        }
+    },
+    .paths = .{
```

Comment:
The example isn't supposed to be a package that other people use so we should just leave paths empty `.paths = {}`.

Reply:
an empty string does the some thing i believe, zig is not happy when you leave the dictionary empty

Reply:
Nevermind then, whoops.